README.md

```diff
@@ -118,9 +110,37 @@ To maximize feedback during a sprint, we've picked a set of "working hours" duri
 09:00-12:00 PDT  daily
 ```
 
-### Active Contributors
+### Roles
+
+#### The Sprint Master
```

Comment:
Who is this currently, you @RichardLitt ? maybe make a note about that somewhere?